src/pages/[l1_l2]/[topic]/[...slug].tsx

```diff
@@ -238,6 +246,8 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
     */
 
     const pathComponents = [l1_l2, topic].concat(slug);
+
+    // const crumbs = await getBreadcrumbsFromSlug(pathComponents.join('/'));
```

Comment:
Just noting that this line is commented out. Do we not need it?

Reply:
I think I left it in there in case we need to put breadcrumbs on this page (they're not in the designs now), but if that ever happens it won't be hard to implement, so I'll take this out.